modules/cloud-router-connection/variables.tf

```diff
@@ -0,0 +1,130 @@
+  type        = string
+}
+variable "fcr_uuid" {
+  description = "Equinix-assigned Fabric Cloud Router identifier"
+  type        = string
+}
+variable "zside_port_name" {
+  description = "Equinix Zside Port Name"
+  type        = string
+  default     = ""
+}
+variable "zside_ap_authentication_key" {
+  description = "Authentication key for provider based connections"
+  type        = string
+}
+variable "zside_ap_type" {
+  description = "Access point type - COLO, VD, VG, SP, IGW, SUBNET, GW"
+  type        = string
+  default     = "SP"
+}
+variable "zside_ap_profile_type" {
+  description = "Service profile type - L2_PROFILE, L3_PROFILE, ECIA_PROFILE, ECMC_PROFILE"
+  type        = string
+  default     = "L2_PROFILE"
+}
+variable "zside_location" {
+  description = "Access point metro code"
+  type        = string
+  default     = "SP"
+}
+variable "zside_link_protocol_type" {
+  description = "Access point protocol type - UNTAGGED, DOT1Q, QINQ, EVPN_VXLAN"
+  type        = string
+  default     = "DOT1Q"
+}
+variable "zside_vlan_tag" {
+  description = "Access point protocol Vlan tag number for DOT1Q connections"
+  default     = ""
+}
+variable "zside_vlan_stag" {
+  description = "Access point protocol Vlan stag number for QINQ connections"
+  default     = ""
+}
+variable "zside_vlan_ctag" {
+  description = "Access point protocol Vlan ctag number for QINQ connections"
+  default     = ""
+}
+variable "peering_type" {
+  description = "Access point peering type - PRIVATE, MICROSOFT, PUBLIC, MANUAL"
+  default     = "PRIVATE"
+}
+variable "network_uuid" {
+  description = "Network UUID"
+  default     = ""
+}
+variable "fabric_sp_name" {
+  description = "Equinix Service Profile Name"
+  type        = string
+}
+variable "seller_region" {
+  description = "Access point seller region"
+  type        = string
+}
+variable "access_key" {
+  description = "AWS Access Key from the AWS Console"
+  type        = string
+  default     = ""
+}
+variable "secret_key" {
+  description = "AWS Secret Key from the AWS Console"
+  type        = string
+  default     = ""
+}
+variable "secondary_connection_name" {
+  description = "Secondary Connection name"
+  type        = string
+  default     = ""
+}
+variable "secondary_redundancy" {
+  description = "Redundancy Priority for the Secondary connection"
+  type        = string
+  default     = "SECONDARY"
+}
+variable "secondary_purchase_order_number" {
+  description = "Purchase order number"
+  type        = string
+  default     = ""
+}
+variable "sec_fcr_uuid" {
+  description = "Equinix-assigned Fabric Cloud Router identifier for Secondary Connection"
+  type        = string
+  default     = ""
+}
+variable "additional_info" {
+  description = "Additional parameters required for some service profiles. It should be a list of maps containing 'name' and 'value  e.g. `[{ name='asn' value = '65000'}, { name='ip' value = '192.168.0.1'}]`"
```

Comment:
The map items are `key` and `value`, but the description says `name` and `value`. Description needs to be updated.